pkg/ui/src/components/tooltip/tooltip.stories.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs, text, boolean, select } from "@storybook/addon-knobs";
+
+import { Tooltip } from "./index";
+import { styledWrapper } from "src/util/decorators";
+
+storiesOf("Tooltip", module)
+  .addDecorator(styledWrapper({ padding: "150px", textAlign: "center" }))
+  .addDecorator(withKnobs)
+  .add(
+    "withKnobs",
+    () => (
+      <Tooltip
+        visible={boolean("Visible", true)}
```

Comment:
`<Tooltip>` component inherits and propagates `defaultVisible` prop to AntTooltip. (see comment below)
I suggest using `defaultVisible` to avoid confusion in names because it also has another `visible` prop.